CMakeLists.txt

```
@@ -285,13 +352,65 @@ endif()
 #)
 
 
+# pip helpers for the impactx package #########################################
+#
+if(ImpactX_PYTHON)
+    set(PYINSTALLOPTIONS "" CACHE STRING
+        "Additional parameters to pass to `pip install`")
+
+    # build the wheel by re-using the lib library we build
+    add_custom_target(pip_wheel
+        ${CMAKE_COMMAND} -E rm -f "impactx*.whl"
+        COMMAND
+            ${CMAKE_COMMAND} -E env PYIMPACTX_LIBDIR=$<TARGET_FILE_DIR:pyImpactX>
+                ${Python_EXECUTABLE} -m pip wheel -v --no-build-isolation --use-feature=in-tree-build ${ImpactX_SOURCE_DIR}
```

Re: https://github.com/ECP-WarpX/WarpX/pull/3106

```suggestion
                ${Python_EXECUTABLE} -m pip wheel -v --no-build-isolation ${ImpactX_SOURCE_DIR}
```